### ISSUE TITLE
License

### ISSUE THREAD

**Issue by acrogenesis** [kungfu-king-betty/mch#1]
Hey, I noticed you forked my macchanger project with some great additions but now you've created it as your own. That's against the MIT license and GitHub's TOS

**Comment by kungfu-king-betty** [comment]
You are correct, I apologize, I was using these repos to experiment with homebrew publishing and maintaining. I will remove these repos.
